src/app/models/annotated-platewellids.ts

```diff
@@ -0,0 +1,3 @@
+export interface AnnotatedPlatewellids {
+  result: Array<{'jnjs': string, 'id': string, 'smiles': string, 'zhang': number}>;
```

Comment:
Let's discuss this over a call. You're missing the point of what a Model should be: "the model is a representation of the unit object"